CHANGELOG.md

```diff
@@ -1,9 +1,8 @@
+<!-- This file is autogenerated. See CONTRIBUTING.md for instructions to add an entry. -->
```

Comment:
```suggestion
<!-- #####################################################
     THIS FILE IS AUTOGENERATED. DO NOT MODIFY MANUALLY!
     See CONTRIBUTING.md for instructions to add an entry.
     ##################################################### -->
```
Not a fan of yelling usually but I think this could be made more prominent.

Reply:
Agreed!